src/test/playground.test.ts

```diff
@@ -77,19 +80,83 @@ describe.skipIf(!playgroundTrue())("playground", () => {
     testParams = getTestParamsForTestnet(publicClient)
   })
 
-  test("should init the smart account", async () => {
+  test("should send a token paymaster user op", async () => {
```

Comment:
Why remove the old test? If this test is important can you keep it in cicd? The playground doesn't run in cicd